frontend/src/store/authSlice.ts

```diff
@@ -17,10 +19,13 @@ export const authSlice = createSlice({
 		setAccessToken: (state, action: PayloadAction<string | null>) => {
 			state.accessToken = action.payload;
 		},
+		setRefreshToken(state, action: PayloadAction<string | null>) {
+			state.refreshToken = action.payload;
+		},
 	},
```

Comment:
How about add `logout` function here?

In many other files, we create logout logic.